src/components/HydrantListItem.js

```diff
@@ -7,6 +7,10 @@ import Collapse from 'material-ui/transitions/Collapse';
 import { FormControl, FormHelperText } from 'material-ui/Form';
 import { withStyles } from 'material-ui/styles';
 import _ from 'lodash';
+import { getElevations } from '../utils/bulkUpdateUtils';
+import {Button} from 'material-ui';
+import OperationMessage from './OperationMessage';
```

Comment:
I'm not a big fan of inserting separate OperationMessages everywhere, since it's really one conceptual component. Just looking around, [react-alert](https://github.com/schiehll/react-alert) seems like a solid package to do what we want, just adds a function prop to add messages that can stack, etc, and pretty easy implementation.